Changelog.md

```diff
@@ -12,6 +12,7 @@
 
 - Remove `activerecord-session_store` gem (#7517)
 - Upgrade to Rails 8 (#7504)
+- Tests added for #new and #create actions for CriteriaController (#7521)
```

Comment:
For consistency with other entries, phrase this as "Add tests for..." (i.e., starting with a verb in the imperative tense).

Also use backticks to surround all code names (`#new`, `#create`, `CriteriaController`).